docs/lake/design.md

```diff
@@ -358,11 +359,15 @@ lake making it easy to measure, tune, and adjust lake parameters to
 optimize layout for performance.
 
 These structures are introspected using meta-queries that simply
-specify a metadata source using a bracket syntax in the `from` operator.
+specify a metadata source using an extended syntax in the `from` operator.
```

Comment:
I remember seeing the "bracket syntax" in some earlier revisions, but it looks like it's gone in favor of the colon/at syntax. My choice of the word "extended" was kind of arbitrary, so please propose a correction if you'd prefer something else!